04-FirstLab/README.md

```diff
@@ -49,5 +49,5 @@ Date:   Thu Mar 17 12:56:25 2016 +0100
 
 # Comments
 
-* When adding content of folder you do not have to add individual files.
+* When adding content of a folder, you do not have to add individual files.
```

Comment:
I meant to comment that sometimes "folder" is used, and sometimes "directory" is used.  Might try to choose one and stick with it.